<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readerwindow</class>
 <widget class="QDialog" name="readerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1446</width>
    <height>685</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>readerWindow</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1451</width>
     <height>681</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>160</width>
       <height>461</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>159</green>
           <blue>207</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>173</red>
           <green>127</green>
           <blue>168</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>159</green>
           <blue>207</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>173</red>
           <green>127</green>
           <blue>168</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>159</green>
           <blue>207</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>159</green>
           <blue>207</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>159</green>
           <blue>207</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>159</green>
           <blue>207</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="scifi_2">
        <property name="text">
         <string>Sciennce-fiction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Romance_2">
        <property name="text">
         <string>Romance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Horror_2">
        <property name="text">
         <string>Horror</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Thriller_2">
        <property name="text">
         <string>Thriller</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="History_2">
        <property name="text">
         <string>History</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clildb_2">
        <property name="text">
         <string>Child books</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Fantasy_2">
        <property name="text">
         <string>Fantasy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Fiction_2">
        <property name="text">
         <string>Fiction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="book">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>591</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 italic 9pt &quot;Noto Sans&quot;;
selection-color: rgb(85, 87, 83);</string>
     </property>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Author    </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Genre     </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Rating        </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Publisher    </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Year    </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Pic</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Status</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="cabinet">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>10</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>cabinet</string>
     </property>
    </widget>
    <widget class="QPushButton" name="serachBtn">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>50</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="piclbl">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>160</y>
       <width>231</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="comments">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>680</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>comments</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLE">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>581</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="placeholderText">
      <string>Lish bi ne na dopku</string>
     </property>
    </widget>
    <widget class="QPushButton" name="orderbtn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>640</y>
       <width>591</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Order selected books</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>169</y>
       <width>231</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="namel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="autl">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Genre:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="genrel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Rating:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="ratl">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Publisher</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="publ">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="yearl">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="loginl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Here you can change your personal data</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>1201</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="NameLE"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Surname:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="surn"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="cinform">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>130</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="erdered">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>561</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 italic 9pt &quot;Noto Sans&quot;;
selection-color: rgb(85, 87, 83);</string>
     </property>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>start</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>end</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>bookId</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>userId</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Go back</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>250</y>
       <width>331</width>
       <height>112</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="nameL">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="autL">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="status">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="pic1">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>350</y>
       <width>321</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
